src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTab.cmp

```diff
@@ -81,6 +82,24 @@
         </div>
     </header>
 
+    <aura:if isTrue="{!v.showRetirementPrompt}">
+        <c:utilPrompt 
+            aura:id="retirementPrompt" 
+            id="retirementPrompt"
+            headerTitle="Batch Gift Entry is retiring"
+            promptMessage="On July 18, 2021, legacy Batch Gift Entry will be retired and will no longer work in NPSP. 
```

Comment:
Did we decide to go with a label for this to be translatable?  Or just have it be text?  I know there was a discussion on the WI but wanted to double check the decision there.

Reply:
I don't know if a decision was made but I think a label makes the most sense, even if the number of non-english customers is ~3. I'll get this PR updated with that.